app/code/Magento/GoogleAnalytics/Test/Unit/Helper/DataTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\GoogleAnalytics\Test\Unit\Helper;
+
+use Magento\GoogleAnalytics\Helper\Data as HelperData;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * Unit test for Magento\GoogleAnalytics\Helper\Data
+ */
+class DataTest extends TestCase
+{
+    /**
+     * @var HelperData
+     */
+    private $helper;
+
+    /**
+     * @var ScopeConfigInterface|MockObject
+     */
+    private $scopeConfigMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp(): void
+    {
+        $this->scopeConfigMock = $this->getMockBuilder(ScopeConfigInterface::class)
+            ->setMethods(['getValue', 'isSetFlag'])
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $objectManager = new ObjectManager($this);
+        $this->helper = $objectManager->getObject(
+            HelperData::class,
+            [
+                'scopeConfig' => $this->scopeConfigMock
+            ]
+        );
+    }
+
+    /**
+     * Test for isGoogleAnalyticsAvailable()
+     *
+     * @return void
+     * @dataProvider gaDataProvider
+     */
+    public function testIsGoogleAnalyticsAvailable($value, $flag): void
+    {
+        $this->scopeConfigMock->expects($this->once())
+            ->method('getValue')
+            ->with(HelperData::XML_PATH_ACCOUNT, ScopeInterface::SCOPE_STORE)
+            ->willReturn($value);
+
+        $this->scopeConfigMock->expects($this->any())
+            ->method('isSetFlag')
+            ->with(HelperData::XML_PATH_ACTIVE, ScopeInterface::SCOPE_STORE)
+            ->willReturn($flag);
+
+        $this->assertEquals(($value && $flag), $this->helper->isGoogleAnalyticsAvailable());
```

Comment:
Please replace this operation `($value && $flag)` with the exat expected result value provided by `gaDataProvider `.

e.g.
```
return [
            ['GA-XXXX', true, true],
            ['GA-XXXX', false, false],
            ['', true, false]
        ];
```

Reply:
@dmytro-ch Thanks for the feedback. I have updated the changes. please review now